src/core/queryObserver.ts

```diff
@@ -208,6 +215,10 @@ export class QueryObserver<
     return this.currentResult
   }
 
+  getTrackedResult(): QueryObserverResult<TData, TError> {
```

Comment:
Maybe name these `getTrackedCurrentResult` as it is the current result but then tracked?

Reply:
see: 6a93c803